lib/gax.js

```diff
@@ -478,13 +500,15 @@ function mergeRetryOptions(retry, overrides) {
  *   in this dictionary.
  * @param {Object} pageDescriptors - A dictionary of method names to
  *   {@link PageDescriptor} objects for methods that are page streaming-enabled.
+ * @param {Object} otherArgs - the arguments to be passed to the API calls.
```

Comment:
Maybe "the non-request arguments..." to be clear that otherArgs doesn't include the request object?